README.md

```diff
@@ -1,9 +1,9 @@
 # Math Magicians
 
```

Comment:
- [ ] Please your readme file is missing some vital information, like cloning and running your app locally. Please checkout this [template](https://github.com/microverseinc/readme-template) and adapt it to your project, as this will make your readme file more informative and professional.

Reply:
Thank you very much for pointing that out